.github/workflows/react-native-ci.yml

```diff
@@ -0,0 +1,61 @@
+name: React Native Frontend CI
+
+
+on:
+  push:
+    branches:
+      - MaryJabro  # Temporary for testing: Adjust to main before pushing to main
```

Comment:
oops looks like you missed this. need to change this to main

Reply:
> oops looks like you missed this. need to change this to main

updating it to main right now, had it as MaryJabro for testing